pyro/contrib/epidemiology/compartmental.py

```diff
@@ -301,14 +301,23 @@ def fit(self, **options):
             that computational cost is exponential in `num_quant_bins`.
             Defaults to 4.
         :param bool haar: Whether to use a Haar wavelet reparameterizer.
+        :param int haar_full_mass: Number of low frequency Haar components to
```

Comment:
is it better to subsume the bool arg entirely into the int arg?

Reply:
That seemed weird, since to trigger `haar` without full mass you would need to pass `haar=0`, so we'd need another distinct value to not use `haar` like `haar=-1`. At that point it is weird enough I'd rather simply expose two options.

Also we may some day add other reparameterizers and rename `haar_full_mass` to `low_freq_full_mass` or something.

Reply:
Do you want me to rename it now? Maybe to `lowpass_full_mass`?

Reply:
probably unnecessary